straxen/plugins/peak_processing.py

```diff
@@ -14,7 +14,11 @@
 @export
 @strax.takes_config(
     strax.Option('n_top_pmts', default=straxen.n_top_pmts,
-                 help="Number of top PMTs"))
+                 help="Number of top PMTs"),
+    strax.Option('check_peak_sum_area', default=True, track=False,
+                 help="Check if the sum area and the sum of area per "
+                      "channel are the same."),
```

Comment:
Just to be sure, you would like to test during processing too and not only during the testing.

Reply:
This would indeed be the idea. However, perhaps it is better to run it once now on a couple of runs to not waste computing it for every single run?

Reply:
@WenzDaniel , in da4a0f1 I've set the default to None, and made the `rtol` configurable. I noticed with processing that I needed for a random sample of runs (in range 11000 - 19000) that I needed a rtol between 2e-4 and 1e-3 to let all the runs complete. Peaks (just a few runs) that failed at an rtol ~= 1e-4 where mostly in the order of `10***7` PE.

I checked as a sanity check if the saturation correction could be involved but it was not (at least disabling made no difference).